atspi-common/src/macros.rs

```diff
@@ -526,6 +552,8 @@ macro_rules! zbus_message_test_case {
 		#[cfg(feature = "zbus")]
 		#[test]
 		fn zbus_msg_conversion_validated_message_with_body() -> () {
+			use crate::events::MessageConversion;
```

Comment:
Local, absolute imports instead of conditionally enabling imports in `event_test_cases!`. I like it!